benchmark/kvbench/test/storage_backend.py

```
@@ -0,0 +1,570 @@
+                    break
+                buf_slice = buffer[offset : offset + size]
+                xfer = self._create_handle(
+                    "WRITE", fd, write_offset + offset, size, buf_slice
+                )
+                handles.append(xfer)
+            return handles
+
+    def close(self):
+        """Close all files and deregister from NIXL."""
+        # Deregister from NIXL
+        for file_path, reg_descs in self._file_reg_descs.items():
```

Ruff (0.15.14)

[warning] Loop control variable `file_path` not used within loop body

Rename unused `file_path` to `_file_path`

(B007)